sale_coupon_criteria_multi_product/__manifest__.py

```diff
@@ -10,6 +10,5 @@
     "author": "Tecnativa, Odoo Community Association (OCA)",
     "maintainers": ["chienandalu"],
     "license": "AGPL-3",
-    "depends": ["sale_coupon"],
-    "data": ["views/sale_coupon_program_views.xml", "security/ir.model.access.csv"],
+    "depends": ["coupon_criteria_multi_product"],
```

Comment:
It should depend on `sale_coupon` as well, don't you think?
```suggestion
    "depends": ["coupon_criteria_multi_product", "sale_coupon"],
```